app/helpers/injection_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module InjectionHelper
   def inject_enterprises
-    inject_json_ams "enterprises", Enterprise.activated.includes(:address).all, Api::EnterpriseSerializer, enterprise_injection_data
+    inject_json_ams "enterprises", Enterprise.activated.includes(address: :state).all, Api::EnterpriseSerializer, enterprise_injection_data
```

Comment:
Interesting, does this have a positive impact on performance?

I can't see any difference in the output.